### ISSUE TITLE
[Suggestions] quickly add multiple pictures

### ISSUE THREAD

**Issue by RolandTi** [bludit/bludit#992]
### Describe your problem
If you want to make a gallery with 10/20 pictures, you can quickly upload them with the bludit media manager, but if you click on "insert", the pop-up close immediately. So it's time consuming to re-open it for every picture.

I don't know if the best behavior is to keep the pop-up open or using checkbox to have the choice to insert multiple pictures at once (or doing nothing ^^').

Thanks for reading this!

### Bludit version
Last one.

**Comment by dignajar** [comment]
Yes sound good, maybe some check boxes on the left side of the image.
Something to improve in the Media Manager.

**Comment by RolandTi** [comment]
Nice, thanks (:
